.pipelines/pr-e2e.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
   - script: make bootstrap
     displayName: Install dependencies
     workingDirectory: $(modulePath)
-  - script: make validate-commit-msg validate-copyright-headers test-style
```

Comment:
should we also remove the script in the Makefile since we're not using it anymore?

Reply:
I'd still like to have a way to validate commit messages locally, and IMHO this still works fine for that case. I'll probably use it, but I can rip it out if we'd rather not have the clutter.